shared/AppInsightsCore/src/JavaScriptSDK/RandomHelper.ts

```diff
@@ -52,12 +52,14 @@ export function randomValue(maxValue: number) {
  * @param signed - True to return a signed 32-bit number (-0x80000000..0x7FFFFFFF) otherwise an unsigned one (0x000000..0xFFFFFFFF)
  */
 export function random32(signed?: boolean) {
-    let value;
+    let value = 0;
```

Comment:
Update to handle edge cases where the built in random functions can return zero when excessively called. So in this case this will fall back to the none hardware methods.